controller_manager/src/controller_manager.cpp

```diff
@@ -1146,7 +1146,7 @@ std::vector<std::string> ControllerManager::get_controller_names()
 
 void ControllerManager::read() { resource_manager_->read(); }
 
-controller_interface::return_type ControllerManager::update()
+controller_interface::return_type ControllerManager::update(const rclcpp::Time& time, const rclcpp::Duration& period)
```

Comment:
Maybe we should provide only time in this method and period is calculated just before controller-update is called. What do you think?

Reply:
Would that still work with `node->now()` from your comment below to get a good `dt` or do you think it's ok to add a couple of variables there to track `dt` using `steady_clock`?

Reply:
@destogl the purpose of this API is to provide both as both should be available from the CM time without having to re-compute the same stuff all the time.